tests/testsuite_basic/conftest.py

```diff
@@ -221,6 +238,29 @@ def distgit_and_remote(tmpdir) -> Tuple[Path, Path]:
     return d, d_remote_path
 
 
+@pytest.fixture()
+def distgit_spec_not_in_root(tmpdir) -> Path:
```

Comment:
I'm sorry to say this but we should remove this fixture because this is not a valid use case: in dist-git, the spec has to be in root all the time, that's how all the packaging tools find it.